paddle/fluid/operators/activation_op.cc

```diff
@@ -93,6 +93,7 @@ framework::OpKernelType GetKernelType(const framework::ExecutionContext& ctx,
                                       const std::string& name) {
   framework::LibraryType library{framework::LibraryType::kPlain};
   framework::DataLayout layout = framework::DataLayout::kAnyLayout;
+  auto data_type = oper.IndicateVarDataType(ctx, name);
```

Comment:
> AFAIK the resulting log from the benchmark machine has checks about GPU ops, in the log there is no oneDNN verbose info, so it looks that oneDNN kernels are not run, which would eventually be correlated with the PR.
Could you advise on that, please?

Although op-benchmark-ci checks about GPU ops, does line 96 have additional time cost? How about move it into line 109?
```
if (library == framework::LibraryType::kPlain && it != oper.Attrs().end()) {
  auto data_type = oper.IndicateVarDataType(ctx, name);
  if (oper.CanMKLDNNBeUsed(ctx, data_type)) {
    xxxx
  }
}
```

Reply:
@luotao1 Thank you for the comment. The `data_type` variable is used also in line 119:
 return framework::OpKernelType(`data_type`, ctx.GetPlace(), layout, library);

 So it will be needed/calculated at the end of the function, whether the condition you mentioned is True or not (despite mkldnn is to be used or not).
 
 In the way it's implemented in PR, the variable value is calculated **only once** per function as it was prior to my changes, without additional time cost.
 It applies to every occurrence in other op files.

Reply:
@wojtuss Thank you for the comment. Having in mind baby sitting the PR-CI-OP-benchmark for more than a week on the same code, I prefer to not refactor the code in the PR. Of course if it's OK with you because I respect you opinion.

Reply:
> In the way it's implemented in PR, the variable value is calculated only once per function as it was prior to my changes, without additional time cost.

Got it.

>  I prefer to not refactor the code in the PR. Of course if it's OK with you because I respect you opinion.

@wojtuss What's your opinion?

Reply:
Totally understandable.